src/test/java/subway/SectionFactory.java

```diff
@@ -0,0 +1,38 @@
+package subway;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.springframework.http.MediaType;
+import subway.line.LineResponse;
+import subway.section.SectionResponse;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SectionFactory {
+
+    public static ExtractableResponse<Response> createSection(Long lineId, Long upstationId, Long downstationId) {
```

Comment:
개인적인 의견이지만, 메서드명을 한글로 더 명확하게 표현되어도 좋을 것 같습니다. 
아래와 같이 네이밍을 하면 테스트코드를 읽을 때, 더 명확하게 표현되고 이해할 수 있을 것 같습니다.
참고부탁드립니다 🙏🏻

```suggestion
    public static ExtractableResponse<Response> 노선에_구간을_추가한다(Long lineId, Long upstationId, Long downstationId) {
```